src/components/PageContainer/PageContainer.tsx

```diff
@@ -2,7 +2,7 @@ import { Button, ButtonColor, ButtonSize, ButtonVariant } from '@digdir/design-s
 import * as React from 'react';
```

Comment:
The line above this has a lot of warnings 😅
Now that DS Button is no longer used here, the imports for it can be removed.